src/utils.tsx

```diff
@@ -2,6 +2,7 @@ import { useState, useEffect, Dispatch, SetStateAction } from "react";
 
 import format from "date-fns/format";
 import formatDistance from "date-fns/formatDistance";
+import moment from "moment";
```

Comment:
I think "moment" used to be a dependency but was removed later on. Why is using moment necessary again?

Reply:
The function timeOfDayFromDate(), which converts a date object to a string, didn't use a official library and converted the dates "manually". This caused some bugs with the formatting, so I had to fix it to make the tests work. I decided to use moment since I have previous experience with the library, and since the already imported library 'date-fns' gave errors.

I have now looked more closely into the error with 'date-fns', and fixed it by adding a extra if-statement to check that the date object passed to the function is valid. Will push a new commit soon were I remove moment and switch to date-fns instead.

Reply:
Good, multiple explicitly imported thid-party libraries to do the same thing is a bit silly.